src/dispatch/incident_cost_type/models.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy_utils import TSVectorType, JSONType
+
+from dispatch.database import Base
+from dispatch.models import DispatchBase, TimeStampMixin
+
+
+# SQLAlchemy Model
+class IncidentCostType(Base, TimeStampMixin):
+    # columns
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    description = Column(String, nullable=False)
+    details = Column(JSONType, nullable=True)
```

Comment:
Added this field in case we want to store links to internal/ external docs describing this type of cost, or other things.